### DISCUSSION TITLE
Can this be ported to MicroPython?

### DISCUSSION THREAD

**Discussion by DanShinde** [gijzelaerr/python-snap7#488] [Ideas]
Just want to know if anyone know a way to port this library to micropython?

**Comment by spreeker** [comment]
Well if ctypes is supported on Micropython then yes should not be so hard
to make a library that works on Micropython.

A quick Google showed me there is no complete implementation of ctypes on
Micropython.


Stephan

**Comment by gijzelaerr** [comment]
Closing as answered. Python-snap7 3.0 is now a pure Python implementation (no ctypes dependency on the native C library), which may make MicroPython porting more feasible as a community effort.
